### ISSUE TITLE
Automatic build process with asset generation fails

### ISSUE THREAD

**Issue by g-voelker** [s4f-dortmund/regionalgruppen_logos#23]
Tried to add Bremen after being requested. The automatic CI build failed. See pull request:
https://github.com/s4f-dortmund/regionalgruppen_logos/pull/22

As a workaround the logos can still be to be build locally.

**Comment by maxnoe** [comment]
I was hoping this just goes away with a texlive update... I cannot reproduce this error even in a fresh environemnt, so I am kind of out of options on how to debug this.

**Comment by maxnoe** [comment]
See #21

**Comment by g-voelker** [comment]
The build process cannot find the font. The problem is therefore most likely related to the automatic build chain. The explicit error is:

```
! Package fontspec Error: The font "Montserrat" cannot be found.

For immediate help type H <return>.
 ...                                              
                                                  
l.4 \newfontfamily
                \caps{Montserrat}[Kerning=Uppercase]

```
I can confirm that this does not occur on my system (arch) with the relevant packages:
linux-lts 5.15 
montserrat-ttf 7.222.1
texlive-core 2022.63035-1
inkscape 1.2-2

**Comment by maxnoe** [comment]
@g-voelker yes, I know, but all tools inside the build say the font is installed: 


```
$ fc-list Montserrat
 /home/runner/.local/share/fonts/static/Montserrat-SemiBold.ttf: Montserrat,Montserrat SemiBold:style=SemiBold,Regular
/home/runner/.local/share/fonts/static/Montserrat-BoldItalic.ttf: Montserrat:style=Bold Italic
/home/runner/.local/share/fonts/static/Montserrat-Medium.ttf: Montserrat,Montserrat Medium:style=Medium,Regular
/home/runner/.local/share/fonts/static/Montserrat-BlackItalic.ttf: Montserrat,Montserrat Black:style=Black Italic,Italic
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Light
/home/runner/.local/share/fonts/static/Montserrat-Light.ttf: Montserrat,Montserrat Light:style=Light,Regular
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Medium
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=SemiBold
/home/runner/.local/share/fonts/static/Montserrat-MediumItalic.ttf: Montserrat,Montserrat Medium:style=Medium Italic,Italic
/home/runner/.local/share/fonts/static/Montserrat-Black.ttf: Montserrat,Montserrat Black:style=Black,Regular
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=ExtraLight
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Bold Italic
/home/runner/.local/share/fonts/static/Montserrat-Bold.ttf: Montserrat:style=Bold
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Regular
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=SemiBold Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Italic
/home/runner/.local/share/fonts/static/Montserrat-Italic.ttf: Montserrat:style=Italic
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Bold
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Thin,Regular
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin
/home/runner/.local/share/fonts/static/Montserrat-Thin.ttf: Montserrat,Montserrat Thin:style=Thin,Regular
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Black
/home/runner/.local/share/fonts/static/Montserrat-ExtraBold.ttf: Montserrat,Montserrat ExtraBold:style=ExtraBold,Regular
/home/runner/.local/share/fonts/static/Montserrat-ThinItalic.ttf: Montserrat,Montserrat Thin:style=Thin Italic,Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin
/home/runner/.local/share/fonts/static/Montserrat-LightItalic.ttf: Montserrat,Montserrat Light:style=Light Italic,Italic
/home/runner/.local/share/fonts/static/Montserrat-ExtraBoldItalic.ttf: Montserrat,Montserrat ExtraBold:style=ExtraBold Italic,Italic
/home/runner/.local/share/fonts/static/Montserrat-Regular.ttf: Montserrat:style=Regular
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Black Italic
/home/runner/.local/share/fonts/static/Montserrat-ExtraLightItalic.ttf: Montserrat,Montserrat ExtraLight:style=ExtraLight Italic,Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Thin Italic,Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Light Italic
/home/runner/.local/share/fonts/Montserrat-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=ExtraBold
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=ExtraLight Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=Medium Italic
/home/runner/.local/share/fonts/Montserrat-Italic-VariableFont_wght.ttf: Montserrat,Montserrat Thin:style=ExtraBold Italic
/home/runner/.local/share/fonts/static/Montserrat-ExtraLight.ttf: Montserrat,Montserrat ExtraLight:style=ExtraLight,Regular
/home/runner/.local/share/fonts/static/Montserrat-SemiBoldItalic.ttf: Montserrat,Montserrat SemiBold:style=SemiBold Italic,Italic
$ luaotfload --find='Montserrat'
luaotfload | resolve : Font "Montserrat" found!
luaotfload | resolve : Resolved file name "/home/runner/.local/share/fonts/static/Montserrat-Regular.ttf"
$ otfinfo -a /home/runner/.local/share/fonts/static/Montserrat-Regular.ttf
Montserrat
```

From https://github.com/s4f-dortmund/regionalgruppen_logos/runs/6082260726?check_suite_focus=true

Event luaotfload-tool finds it, which is the system used by lualatex to find fonts...

**Comment by maxnoe** [comment]
I filed this issue with fontspec here:
https://github.com/latex3/fontspec/issues/459

**Comment by g-voelker** [comment]
Your script loads the otf font. Did you try using the ttf font? I just updated my fork, will look into it when I have a little time.

**Comment by maxnoe** [comment]
I don't explicitly load the ttf or the otf font, the system should look for both...

**Comment by maxnoe** [comment]
Fixed  by #24
